scripts/substrate-diff.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Helper script which outputs the diff to Substrate's `node-template`.
+SUBSTRATE_DIR=${SUBSTRATE_DIR:-~/projects/substrate}
```

Comment:
I'd change this to be user provided since not everyone has their Substrate build in `~/projects/substrate` 😉

Reply:
It already does this, this is just the default. The bash syntax is `${VAR:-default}`.

So you can invoke the script with `SUBSTRATE_DIR=/tmp/substrate substrate-diff.sh`.

Reply:
Yeah, but imo it's not a great default. Having no default and enforcing that users point to their Substrate folder would be better